spec/controllers/miq_alert_controller_spec.rb

```diff
@@ -154,6 +154,36 @@
       expect(assigns(:flash_array)).to be_nil
     end
 
+    it "forces Event to check to be present for Eval type - Event threshold" do
```

Comment:
super minor, but certain security scanners see the word "eval" and freak out (yes, even inside strings 🤦 ).  Can you change the word eval to something else here and below?